scripts/util/graph_memory.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/env bash
```

Comment:
@bwitham here is a script that I came across and then modified that you might be interested in.  It graphs the memory usage of a process while it runs and then displays it.  Just run the following:
`$HOOT_HOME/scripts/util/graph_memory.sh hoot.bin`

And you'll get something like this created.  This is what `hoot alpha-shape ...` looks like:
![graph_memory](https://user-images.githubusercontent.com/13385275/101817742-75510600-3ae8-11eb-9a17-61b04fa8cb6c.png)

Reply:
That is cool. Definitely could make use of that.